src/connectors/menu/connectMenu.js

```diff
@@ -38,6 +38,7 @@ Full documentation available at https://community.algolia.com/instantsearch.js/v
  * @typedef {Object} CustomMenuWidgetOptions
  * @property {string} attribute Name of the attribute for faceting (eg. "free_shipping").
  * @property {number} [limit = 10] How many facets values to retrieve.
+ * @property {boolean} [showMore=false] Whether to display a button that expands the number of items.
```

Comment:
```suggestion
  * @property {boolean} [showMore = false] Whether to display a button that expands the number of items.
```

Reply:
Yeah – this will be removed with the new doc anyway.